jenkins-integration/jenkins-jobs/common/scripts/jenkins/pipeline.groovy

```diff
@@ -403,6 +417,7 @@ def multipass_pipeline(jobs) {
                     SCRIPT_DIR,
                     server_era)
             step030_customize_settings(SCRIPT_DIR, job['puppet_settings'])
+            step035_customize_hocon_settings(SCRIPT_DIR, job['hocon_settings'])
```

Comment:
I don't have a strong opinion one way or the other, so, happy to go with @camlow325 's suggestion of moving this to 80.